scripts/globals/items/death_scythe.lua

```diff
@@ -0,0 +1,37 @@
+-----------------------------------------
+-- ID: 16777
+-- Item: Death Scythe
+-- Additional Effect: Drains HP
+-----------------------------------------
+require("scripts/globals/status");
+
+-----------------------------------
+-- onAdditionalEffect Action
+-----------------------------------
+function onAdditionalEffect(player,target,damage)
+    local chance = 10;
+    if (target:getMainLvl() > player:getMainLvl()) then

```

Comment:
I apologize for the tone-in-text I just gave you..Got annoyed at seeing it yet again.